src/IndieWeb/MentionClient.php

```diff
@@ -282,7 +361,14 @@ public static function findLinksInJSON($input) {
     return $links;
   }
 
-  public function sendMentions($sourceURL, $sourceBody=false) {
+  /**
+   * Tries to send webmention and pingbacks to each link on $sourceURL. Depends on Microformats2
+   * @param $sourceURL string URL to examine to send mentions to
+   * @param bool $sourceBody if true will search for outgoing links with this (string).
+   * @return int
+   * @see \Mf2\parse
```

Comment:
sorry which line? This was a little while ago, I can't recall, i can take it out. I put @see because it made more sense to me after checking mf2 parse.